src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -14,23 +15,28 @@ internal class CoreXT
     {
         public string ConfigFilePath { get; }
         public string ComponentsFilePath { get; }
-        public XDocument Config { get; }
-        public JObject Components { get; }
+        public XDocument Config { get; }        
+        private static Dictionary<string, string> ComponentToFileMap { get; set; }
```

Comment:
Why static?

Reply:
Since it is getting initialized in Load() which is static, this object also needs to be static.